File: VB/UploadControlExample/Models/UploadControlValidationSettings.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports DevExpress.Web

Public Class MyUploadControlValidationSettings
    Public Shared Settings As New UploadControlValidationSettings() With {.AllowedFileExtensions = New String() {
        ".jpg", ".jpeg", ".png", ".bmp", ".pdf", ".xml", ".txt", ".rtf", ".docx", ".xls", ".xlsx"},
        .MaxFileSize = 4194304}
End Class

File: CS/UploadControlExample/Models/UploadControlValidationSettings.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using DevExpress.Web;

public class MyUploadControlValidationSettings {
    public static UploadControlValidationSettings Settings = new UploadControlValidationSettings() {
        AllowedFileExtensions = new string[] { ".jpg", ".jpeg", ".png", ".bmp", ".pdf", ".xml", ".txt", ".rtf", ".docx", ".xls", ".xlsx" },
        MaxFileSize = 4194304
    };
}